cvmfs/receiver/reactor.cc

```diff
@@ -28,7 +28,7 @@ Reactor::Request Reactor::ReadRequest(int fd, std::string* data) {
   using namespace receiver;  // NOLINT
 
   // First, read the command identifier
-  int32_t req_id = 0;
+  int32_t req_id = kQuit;
   int nb = SafeRead(fd, &req_id, 4);
```

Comment:
Let's replace the `4` by `sizeof(req_id)`, and also in the following lines of the code

Reply:
I believe this is 4 to reflect the one on the gateway.

So it will be `4` on both size. Insetad of `sizeof()` on the c++ part and 4 on the go part.

Reply:
Fair enough, let's add this as a code comment then.

Reply:
That it is just a guess!!!

I didn't write this code :laughing: